shell/packages/sandstorm-db/db.js

```diff
@@ -56,6 +55,11 @@ if (Meteor.isServer && process.env.LOG_MONGO_QUERIES) {
 //       verifiedEmail: Only provided by some services. Cannot be directly edited by the user.
 //       main: True if this is the user's main identity.
 //       noLogin: True if the user does not trust this identity for account authentication.
+//       devName: Only present when `service` is "dev". A string containing the name used to create
+//                this dev account.
+//       emailTokens: Only present when `service` is "emailToken". An array of objects of the form
```

Comment:
Proposal: we could make each identity be of the following form:
```
{  "id" : "a1bf13e1da9a356086d8c4fe5d52fd096d9ac32eddf95bec1e1f492e310afc82",
   "service" : {"emailToken": {"email": "dwrenshaw@gmail.com", "tokens": []}},
   "main" : true,
   noLogin: false,
   "profile": {"name" : "David Renshaw",
                  "handle" : "dwrensha",
                  "pronoun" : "male",
                  "verifiedEmail" : "dwrenshaw@gmail.com",
                  "unverifiedEmail" : "david@example.com" }
}
```
 
Then identities.profile could be freely shared with contacts.

If you agree with this proposal, do you think it makes sense to roll it into the present pull request?